client/components/site-selector/index.jsx

```diff
@@ -291,8 +314,11 @@ const SiteSelector = React.createClass( {
 } );
 
 export default connect( ( state ) => {
+	const visibleSiteCount = get( getCurrentUser( state ), 'visible_site_count', 0 );
```

Comment:
The selector is pretty trivial, but still feel `getCurrentUser( state )` could be assigned to a `user` variable to avoid being called twice in this function.